cmd/e2e/build_crashing_test.go

```diff
@@ -40,13 +39,12 @@ var _ = Describe("Crashing Build", func() {
 			g.Expect(verifyGameServerBuildOverall(ctx, kubeClient, state)).To(Succeed())
 		}, 45*time.Second, interval).Should(Succeed()) // bigger timeout because of the time crashes take to occur and captured by the controller
 
-		// we are updating the GameServerBuild to be able to have more crashes for it to become Unhealthy
+		// we are updating the GameServerBuild with a big CrashesToMarkUnhealthy to give time to the GameServerBuild to become Healthy
 		gsb := &mpsv1alpha1.GameServerBuild{}
 		err = kubeClient.Get(ctx, client.ObjectKey{Name: testBuildCrashingName, Namespace: testNamespace}, gsb)
 		Expect(err).ToNot(HaveOccurred())
 		patch := client.MergeFrom(gsb.DeepCopy())
-		gsb.Spec.CrashesToMarkUnhealthy = 10
-
+		gsb.Spec.CrashesToMarkUnhealthy = 1000
```

Comment:
Why is this value set to 1000 but then changed to 10? Are they different tests?

Reply:
at one point during running the e2e tests, I noticed that this test failed. This is because the GameServers crashed multiple (10) times so the GameServerBuild stayed Unhealthy, before having the chance to transition (temporarily) to Healthy. So, bu setting it to 1000 we give it more chance to transition to Healthy, before decreasing it to 10 (in the next step) so that it can become Unhealthy again.
Let me take this and post it as a comment, thanks!